src/TestSuite/Fixture/TestFixture.php

```diff
@@ -146,17 +156,15 @@ protected function _schemaFromReflection(): void
         $db = ConnectionManager::get($this->connection());
         assert($db instanceof Connection);
         try {
-            $name = Inflector::camelize($this->table);
-            $ormTable = $this->fetchTable($name, ['connection' => $db]);
-
-            // Remove the fetched table from the locator to avoid conflicts
-            // with test cases that need to (re)configure the alias.
-            $this->getTableLocator()->remove($name);
+            $ormTable = $this->fetchTable($this->alias, ['connection' => $db]);
```

Comment:
What should the behavior be if the fixture has both `alias` and `table` defined? For example

```php
class AliasedArticlesFixture extends TestFixture
{
    public string $alias = 'Articles';
    public string $table = 'articles_lol';
}
```

Should we be overriding the `table` option for `fetchTable()`?